criticality_score/generate.py

```diff
@@ -92,6 +95,15 @@ def get_github_repo_urls_for_language(urls, sample_size, github_lang=None):
 
 
 def main():
+    # logging
+    log_filename = os.path.join('output', 'generate.log')
```

Comment:
we need to use args.output_dir instead of hardcoded output, it can be passed as arg to initialize_logging_handlers
also, generate.log can be renamed to output.log.

Reply:
Okay, it looks like this now:
https://github.com/ossf/criticality_score/pull/66/files